src/platforms/android/configuration/app-not-respond.mdx

```diff
@@ -4,6 +4,8 @@ sidebar_order: 3
 description: "Learn how to turn off or specify ANR."
 ---
 
+# ANR implementation details
```

Comment:
```suggestion
## ANR Implementation Details
```
It would be great if you could add an introductory phrase after the heading (Application Not Responding (ANR)) so that it's not 2 headings one right after the other. Alternatively, we can make ANR Implementation Details the heading instead if it makes more sense.